.travis.yml

```diff
@@ -12,6 +12,10 @@ matrix:
           python: 3.7
           dist: xenial
           sudo: true
+        - os: linux
+          python: 3.8
+          dist: xenial
```

Comment:
You can leave out `dist: xenial`.

Reply:
Can I remove it from the 3.7 case too?  I was just copying that.

Reply:
It's fine, I've plans of updating the travis.yml in alignment with the other packages I've updated for https://github.com/zopefoundation/meta/issues/11 some time this week anyway.

Reply:
Yes, it is no longer necessary as xenial now is the default environment.
https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment